--- origsrc/oxygen-gtk3-1.2.0/src/oxygenqtsettings.cpp	2013-08-28 08:56:22.000000000 -0500
+++ src/oxygen-gtk3-1.2.0/src/oxygenqtsettings.cpp	2013-09-02 19:09:45.331009600 -0500
@@ -327,7 +327,7 @@ namespace Oxygen
         struct stat st;
         if( stat( _userConfigDir.c_str(), &st ) != 0 )
         {
-            #if _POSIX_C_SOURCE
+            #if !defined(Q_OS_WIN)
             mkdir( _userConfigDir.c_str(), S_IRWXU|S_IRWXG|S_IRWXO );
             #else
             mkdir( _userConfigDir.c_str() );
